OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -24,6 +24,7 @@ public class FindResources : Activity
 		readonly HarvesterInfo harvInfo;
 		readonly Mobile mobile;
 		readonly MobileInfo mobileInfo;
```

Comment:
We should be able to remove some of these cached references after updating `PathGraph` to take the `locomotorInfo`.